src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -45,5 +45,29 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
 
             return builder;
         }
+
+        public static void ConfigureOptions(EventHubOptions options)
+        {
+            string offsetType = options?.InitialOffsetOptions?.Type?.ToLower() ?? String.Empty;
+            if (!offsetType.Equals(String.Empty))
+            {
+                switch (offsetType)
+                {
+                    case "fromstart":
+                        options.EventProcessorOptions.InitialOffsetProvider = (s) => { return EventPosition.FromStart(); };
+                        break;
+                    case "fromend":
+                        options.EventProcessorOptions.InitialOffsetProvider = (s) => { return EventPosition.FromEnd(); };
+                        break;
+                    case "fromenqueuedtime":
+                        DateTime enqueuedTimeUTC = DateTime.Parse(options.InitialOffsetOptions.EnqueuedTimeUTC).ToUniversalTime();
```

Comment:
Can we throw an exception with an user friendly message with expected format where we can not parse the `DateTime`. It would be helpful for the cutomers.

Reply:
Good point.  Addressed.  Exception will now look like this:
'EventHubOptions:InitialOffsetOptions:EnqueuedTimeUTC is configured with an invalid format. Please use a format supported by DateTime.Parse().  e.g. 'yyyy-MM-ddThh:mm:ssZ''
Inner Exception
FormatException: String '2020-11-13T1134508:46:45Z' was not recognized as a valid DateTime.